sql/memo/rel_props.go

```diff
@@ -251,6 +251,7 @@ func allTableCols(rel SourceRel) sql.Schema {
 	sch := projTab.PrimaryKeySchema().Schema
 	ret := make(sql.Schema, len(sch))
 	for i, c := range sch {
+		// TODO: generation_expression
```

Comment:
marking these as virtual might be useful in some cases? can always add later

Reply:
Yeah, need tests first.